.eslintrc.js

```diff
@@ -1,280 +1,214 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'commonjs': true,
-        'es6': true
+    env: {
+        browser: true,
```

Comment:
My personal reason for appreciating dropping quoting object keys that are legal identifiers: it enables syntax highlighting. For me, string syntax highlighting gives me no indication of what the token is. I like being able to tell it's an object key. Not a big deal to me either way though. I just hit format, and move on. Guilty 😜